torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -24,7 +35,17 @@ def get_dimensions(image: Union[features.ImageTypeJIT, features.VideoTypeJIT]) -
         return get_dimensions_image_pil(image)
 
 
-get_num_channels_image_tensor = _FT.get_image_num_channels
+def get_num_channels_image_tensor(image: torch.Tensor) -> int:
```

Comment:
Not sure if I'm missing something here, but can't we use `get_dimensions_image_tensor` from above and just return the channel?

Reply:
Because we do a billion assertions recursively without good reason. The new implementation cleans up the mess.